### DISCUSSION TITLE
Supports for screen readers on Android

### DISCUSSION THREAD

**Discussion by makhlwf** [godotengine/godot-proposals#7234] [Platforms]
Hello, I am using Android
I have poor eyesight, but it is not very severe, but I need to use talkback in interfaces that have elements and small icons, and I found that godot does not support dealing with screen readers
I really hope to support that, even if in a small way, believe me, it will mean a lot to me

**Comment by AThousandShips** [comment]
Related:
* #1964
* #983

This is a really important feature I agree, but it can be hard to add because of how many different platforms we support, it has to work for Windows, macOS, Linux, etc.

There is work being done on getting it to work for some platforms using AccessKit, see here:
* https://github.com/godotengine/godot/pull/76829

(I hope you can access the links above, if not please say and I'll try and add them in a more accessible way)

**Reply by makhlwf** [reply]
thank you for your reply
I wish  success in adding support
lol i was able to open the links thank you

**Reply by AThousandShips** [reply]
Sorry if my question was weird or offensive, not experienced with screen readers myself and unsure how accessible GitHub is :)

**Reply by bruvzg** [reply]
> There is work being done on getting it to work for some platforms using AccessKit, see here

Currently, AccessKit supports desktop platforms only (support for Android, iOS, and Web is planned, but I do not think there's any timeline for it at the moment).
